spiceaidocs/docs/components/data-connectors/sharepoint.md

```diff
@@ -51,66 +51,80 @@ The sharepoint connector does not yet support creating a dataset from a single f
- 1. Service principal authentication, by setting the `sharepoint_client_secret` parameters.
+As outlined in the [connector parameters](#parameters), the SharePoint connector supports two types of authentication:
+ 1. Service principal authentication, by setting the `sharepoint_client_secret` parameter.
  2. User authentication, by setting the `sharepoint_auth_code` parameter. Generally this is obtained by running `spice login sharepoint` and following the OAuth2 flow.
 
 ### Creating an Enterprise Application
-To use the Sharepoint connector with service principal authentication, you will need to create an Azure AD application and grant it the necessary permissions. This will also support OAuth2 authentication for users within the tenant (i.e. `params.sharepoint_auth_code`).
+To use the SharePoint connector with service principal authentication, you will need to create an Azure AD application and grant it the necessary permissions. This will also support OAuth2 authentication for users within the tenant (i.e. `sharepoint_auth_code`).
 
 1. Create a new Azure AD application in the [Azure portal](https://portal.azure.com/#view/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/~/Overview).
 2. Under the application's `API permissions`, add the following permissions: `Sites.Read.All`, `Files.Read.All`, `User.Read`, `GroupMember.Read.All`
-  - For service principal authentication, Application permissions are required. For user authentication, only delegated permissions are required.
-3. Add `sharepoint_client_id` and `sharepoint_tenant_id` to the connector configuration.
-4. (For service principal authentication): Under the application's `Certificates & secrets`, create a new client secret. Use this in `params.sharepoint_client_secret`.
+    - For service principal authentication, Application permissions are required. 
+    - For user authentication, only delegated permissions are required.
+3. Add `sharepoint_client_id` (from the `Application (Client) ID` field) and `sharepoint_tenant_id` to the connector configuration.
+4. (For service principal authentication): Under the application's `Certificates & secrets`, create a new client secret. Use this for the `sharepoint_client_secret` parameter.
 
 ### Default Spice Application
-For convenience, Spice AI maintains a default Azure AD application that can be used for Sharepoint authentication. This application requires OAuth2 authentication. To use it:
+For your convenience, Spice AI maintains a default Entra (Azure AD) application that can be used for authentication against your SharePoint instance. This application requires OAuth2 authentication. To use it:
+
 ```yaml
 datasets:
   - from: sharepoint:me/root # Set the drive and subpath as needed.
     name: my_data
     params:
       sharepoint_client_id: f2b3116e-b4c4-464f-80ec-73cd9d9886b4
-      sharepoint_tenant_id: #{env:TENANT_ID}
+      sharepoint_tenant_id: #${env:TENANT_ID}
```

Comment:
Is the # supposed to be here?

Reply:
It's up to the user to fill that value in from their Tenant